t/lib/t/MusicBrainz/Server/Controller/WS/2/GenreList.pm

```diff
@@ -1,7 +1,7 @@
 package t::MusicBrainz::Server::Controller::WS::2::GenreList;
 use Test::Routine;
 
-use MusicBrainz::Server::Test::WS qw( ws2_test_xml );
+use MusicBrainz::Server::Test::WS qw( ws2_test_xml ws2_test_json );
```

Comment:
You're removing this again in the next commit, but AFAICT it doesn't need adding at all?